task/sql-tasks.js

```diff
@@ -280,7 +498,35 @@ async function task_1_21(db) {
  * @return {array}
  */
 async function task_1_22(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+ 
+    SELECT DISTINCT
+        t_Cust.CompanyName,
+        t_Prod.ProductName,
+        t_OrderD.UnitPrice AS PricePerItem
+    FROM
+        Customers AS t_Cust
+            INNER JOIN
+        Orders AS t_Order ON t_Order.CustomerID = t_Cust.CustomerID
+            INNER JOIN
+        OrderDetails AS t_OrderD ON t_OrderD.OrderID = t_Order.OrderID
+            INNER JOIN
+        Products AS t_Prod ON t_OrderD.ProductID = t_Prod.ProductID
+    WHERE
+        t_OrderD.UnitPrice = (SELECT 

```

Comment:
still slow. Try to group by customer id and join this subquery on price and cutomer id